lib/core/include/irods/connection_pool.hpp

```diff
@@ -128,20 +159,20 @@ namespace irods
                         const std::string& _host,
                         const int _port,
                         experimental::fully_qualified_username _username,
-                        const int _refresh_time);
+                        const connection_pool_options& _options = {});
```

Comment:
Are we not deprecating the old constructors? Can't remember where we landed on this vis a vis deprecation process.

Reply:
Oh okay. Very good